src/mesh/Tetrahedron.cpp

```diff
@@ -34,6 +32,9 @@ Tetrahedron::Tetrahedron(
           (&vertexTwo != &vertexFour) &&
           (&vertexThree != &vertexFour),
       "Tetrahedron vertices are not unique!");
+
+  std::sort(_vertices.begin(), _vertices.end(),
+            [](const Vertex *lhs, const Vertex *rhs) { return *lhs < *rhs; });
```

Comment:
Needs documentation: why do we need sorting?
Do we have a unit test that tests the sorting?

Reply:
We can test this more efficiently in #1313